eve/workers/unit_and_feature_tests/Dockerfile

```diff
@@ -1,20 +1,23 @@
 FROM buildpack-deps:jessie-curl
 
 #
-# Install apt packages needed by backbeat and buildbot_worker
+# Install apt packages needed by utapi and buildbot_worker
 #
 ENV LANG C.UTF-8
+ENV NODE_VERSION 10.22.0
```

Comment:
Are we moving back to v10.22.0 for Utapi?

Had a discussion with @tmacro in one of his older PRs and he wanted to use node 12.
https://github.com/scality/utapi/pull/441#discussion_r430591701 which was targeted for `development/7.7`

Reply:
Discussed and resolved :)

Reply:
For some additional context, Utapi v2 is being written be compatible with 12.x with no changes (in fact its what I use to develop) but given the time constraints I decided that it would take to long to update v1 so I left the deployed version as 10.x